tools/wdio/plugins/custom-commands.mjs

```diff
@@ -105,6 +105,33 @@ export default class CustomCommands {
       })
     })
 
+    /**
+     * Sets a permanent scheduled reply for the rum call to include the session
+     * replay flag with a value of 1 enabling the feature.
+     */
+    browser.addCommand('mockDateResponse', async function (enableSessionReplay) {
+      const serverTime = Date.now() - (60 * 60 * 1000) // Subtract an hour to make testing easier
```

Comment:
This could be an optional passed in parameter instead of always being hard coded to an hour before current time.